reference/functions/int.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: int
+published: true
+tags: [reference, functions, int]
+---
+
+[%CFEngine_function_prototype(string)%]
+
+**Description:** Convert numeric string to int.
+
+[%CFEngine_function_attributes(string)%]
+
+If `string` represents a floating point number then the decimals are *truncated*.
+
+```cfengine3
```

Comment:
Instead of having an inline example, I think we should leverage your the example from core.

Replace all that with:

  `[%CFEngine_include_example(int.cf)%]`